DeepStorage/CompDeepStorage.cs

```diff
@@ -375,10 +376,29 @@ public class CompDeepStorage : ThingComp, IHoldMultipleThings.IHoldMultipleThing
             */
         }
 
+        public override void PostSpawnSetup(bool respawningAfterLoad)
+        {
+            base.PostSpawnSetup(respawningAfterLoad);
+            _storageBuilding = new Deep_Storage_Building(this.parent as Building_Storage);
+        }
+
+        public override void PostDeSpawn(Map map)
+        {
+            base.PostDeSpawn(map);
+            _storageBuilding.Clear();
+        }
+
+        public override void PostDestroy(DestroyMode mode, Map previousMap)
+        {
+            base.PostDestroy(mode, previousMap);
+            _storageBuilding.Clear();
```

Comment:
Why bother to Clear() it, if the building is destroyed?  The comp is going away anyway?

Reply:
Just a habit to clean things up. Removing references to things could help in GC.